packages/runtime/src/cdn/full.ts

```diff
@@ -11,6 +12,9 @@ init({
       presetUno(),
       presetTypography(),
       presetTagify(),
+      presetIcons({
+        cdn: 'https://esm.sh/',
```

Comment:
I am not sure to hard-code this. Since the CDN mode of preset-icons is not self-included, I would recommend asking users to use the bundler approach for their custom needs. That's why we didn't include it by default at the beginning.